README.md

```diff
@@ -4,10 +4,27 @@
 
 Fast compile-time regular expressions with support for matching/searching/capturing during compile-time or runtime.
 
-You can use the single header version from directory `single-header`. This header can be regenerated with `make single-header`. If you are using cmake, you can add this directory as subdirectory and link to target `ctre`.
-
 More info at [compile-time.re](https://compile-time.re/)
 
+ - [What this library can do](#What-this-library-can-do)
+   - [Unicode support](#Unicode-support)
+   - [Unknown character escape behaviour](#Unknown-character-escape-behaviour)
+ - [Supported compilers](#Supported-compilers)
+ - [API Overview](#API-Overview)
+   - [Range outputing API](#Range-outputing-API)
+   - [Functors](#Functors)
+   - [Possible subjects (inputs)](#Possible-subjects-\(inputs\))
+   - [Template UDL syntax](#Template-UDL-syntax)
+   - [C++17 syntax](#C++17-syntax)
+   - [C++20 syntax](#C++20-syntax)
+ - [Examples](#Examples)
+   - [Extracting number from input](#Extracting-number-from-input)
+   - [Extracting-values-from-date](#Extracting-values-from-date)
+   - [Using captures](#Using-captures)
+   - [Lexer](#Lexer)
+   - [Range over input](#Range-over-input)
+   - [Unicode](#Unicode)
+ - [Integration](#Integration)
```

Comment:
ToC is already provided by GitHub's UI.

Reply:
I am using the github.com website that's why I created the ToC

Reply:
Yes, GitHub UI provides a ToC already:

![image](https://user-images.githubusercontent.com/1736896/141704070-4de2cadd-06a5-4451-8120-2063a93d0409.png)

Reply:
Oh I didn't know that